src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/barnetilsyn/VedtakTilsynBarn.kt

```diff
@@ -30,7 +33,9 @@ data class VedtakTilsynBarn(
                 require(beregningsresultat != null) { "Mangler beregningsresultat for type=$type" }
                 require(vedtak != null) { "Mangler vedtak for type=$type" }
             }
+
             TypeVedtak.AVSLAG -> {
+                require(årsakerAvslag != null && årsakerAvslag.årsaker.isNotEmpty()) { "Må velge minst en årsak for avslag" }
```

Comment:
Tror dette er en en greie
```suggestion
                require(!årsakerAvslag.isNullOrEmpty()) { "Må velge minst en årsak for avslag" }
```

Reply:
Tror ikke det funker fordi `årsakerAvslag` er en wrapper rundt `årsaker`. Så enten er `årsakerAvslag = null` eller så skal `årsakerAvslag.årsaker` ha innhold som liste